### DISCUSSION TITLE
First PR to Django is open!

### DISCUSSION THREAD

**Discussion by RealOrangeOne** [RealOrangeOne/django-tasks#116] [Announcements]
The base PR is now open on Django: https://github.com/django/django/pull/18627/.

Specifically, it contains:

- Common API interface
- Base backend
- Task / result classes
- Immediate / Dummy backends
- Documentation

The database backend is intentionally absent, and will be added in future (**after** this PR is merged).

**Comment by washeck** [comment]
@RealOrangeOne What is the status of the PR? It would be awesome to see it merged. Thanks.

**Comment by hmpf** [comment]
It has been merged for Django 6.0, we're now waiting for the release.
